src/components/Card/Card.js

```diff
@@ -1,37 +1,40 @@
-import { useState } from "react";
+import { useState, useContext } from "react";
+import PropTypes from 'prop-types';
 
 import classes from "./Card.module.css";
 import { AiFillEdit } from "react-icons/ai";
 import { AiFillCheckCircle } from "react-icons/ai";
 import { AiFillCloseCircle } from "react-icons/ai";
 import CardHeader from "../CardHeader/CardHeader";
 import CardBody from "../CardBody/CardBody";
+import DataContext from "../../store/data-context";
 
 function Card(props) {
   const [editable, setEditable] = useState(false);
   const [curCaptionValue, setCurCaptionValue] = useState(props.caption);
   const [curTextValue, setCurTextValue] = useState(props.text);
+  const dataCtx = useContext(DataContext);
```

Comment:
You got it right about how does Context work.
But may be it would be good to leave those low-level components (like Card or Button) reusable? Its more difficult to use them in other places when some global context is used.
You may leave it as is if you want, I approve the PR. There will be a Redux task (09.01), and you can do this change later, when the Redux will be used.

Reply:
I think, this is good idea to make them reusable again. Thank you for advice